srcpkgs/wf-config/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'wf-config'
+pkgname=wf-config
+version=0.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config"
+makedepends="libevdev-devel wlroots-devel"
+short_desc="Wayfire library for managing configuration files"
+maintainer="Young Jin Park <youngjinpark20@gmail.com>"
+license="MIT"
+homepage="https://wayfire.org"
+distfiles="https://github.com/WayfireWM/wf-config/archive/v${version}.tar.gz"
```

Comment:
add this line to distfiles, below the current one (inside the double quotes that denote distfiles=.

https://raw.githubusercontent.com/WayfireWM/wf-config/master/LICENSE>LICENSE.txt